charts/camunda-platform/charts/identity/templates/postgresql-configmap.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if or .Values.postgresql.enabled .Values.externalDatabase.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "identity.postgresql.id" . }}
+  labels: {{- include "identity.labels" . | nindent 4 }}
+  annotations: {{- toYaml  .Values.global.annotations | nindent 4 }}
+data:
+  MULTITENANCY_ENABLED: {{ .Values.global.multitenancy.enabled | quote }}
```

Comment:
I put it there because MT doesn't work without PSQL.

In fact, we can rename that configmap and use it for any new config, not only psql or mt (instead of having that in the deployment file).